docs/pages/api.rst

```diff
@@ -57,19 +57,21 @@ Station list
 Get station information for a given set of *parameter*, *time resolution*
 and *period type* options.
 
-.. code-block:: python
+.. ipython:: python
```

Comment:
So, that will kind of give us at least a small part of what `doctest` offers? Sphinx will execute the code and propagate an error when something croaks?

Reply:
My first intention here was only to make results visible to the user as to that one would have an idea how the result looks like. But you're right that this also helps us to catch those errors. I didn't activate the ipython environment for the "storage code block" though as I am unsure how the result will look like or where things are finally stored...